contracts/nft/src/execute/validate.rs

```diff
@@ -0,0 +1,24 @@
+use cosmwasm_std::{Coin, Deps};
+
+use crate::core::error::ContractError;
+use crate::{core::msg::ExecuteMsg, util::validate::Validate};
+
+impl Validate for ExecuteMsg {
+    fn validate(&self, _deps: Deps) -> Result<(), ContractError> {
+        Ok(())
+    }
+
+    fn validate_funds(&self, _funds: &[Coin]) -> Result<(), ContractError> {
```

Comment:
Do all of these messages want users to provide funds?

Reply:
not for this example, it is to be assumed the "purchasing" is OOB since the spec makes no mention of it

Reply:
I only asked because the validation is saying that they are Ok if they provide funds. I guess since it's an example it doesn't matter?